style.css

```diff
@@ -0,0 +1,304 @@
+  padding-left: 20px;
+  text-align: center;
+}
+
+.left-bar .time {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  letter-spacing: -0.28px;
+  color: #fff;
+}
+
+.left-bar .name {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.right-bar {
+  width: 50%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  padding-right: 20px;
+  justify-content: space-around;
+}
+
+.container {
+  position: relative;
+  height: 55%;
+  width: 80%;
+  top: 130px;
+  left: 40px;
+
+  /* overflow: hidden; */
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 10px;
+}
+
+.container .intro {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.container .title {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  color: #fff5e1;
+}
+
+.container .description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.logo {
+  display: flex;
+  align-items: flex-end;
+  gap: 10px;
+}
+
+.logo .linkedin {
+  margin: 0;
+  padding: 0;
+}
+
+.main-image img {
+  position: relative;
+  height: 370px;
+  width: 100px;
+  left: 96.2%;
+  top: -100%;
+}
+
+@media all and (max-width: 1800px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 101%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1700px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 99.7%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1600px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 98.2%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1500px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 96.9%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1400px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 94.2%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1300px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 91.8%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1200px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 88.2%;
+    top: -100%;
+  }
+}
+
+@media all and (max-width: 1100px) {
+  .main-image img {
+    position: relative;
+    height: 370px;
+    width: 100px;
+    left: 85.2%;
+    top: -98%;
+  }
+}
```

Comment:
- [ ] As suggested, please make sure that you avoid using negative values. This is because negative values for percentage in CSS can cause layout issues and unexpected behaviour, as they can result in elements being positioned outside of the containing element. And we don't want that  to happen, right?🎯. For more clarity, please follow the lessons mentioned below
- [Percentage](https://developer.mozilla.org/en-US/docs/Web/CSS/percentage)
- [Position](https://css-tricks.com/almanac/properties/p/position/)